contrib/launch-venv.sh

```
@@ -1,4 +1,10 @@
 #!/usr/bin/env -S bash -e
+
+# NixOS: re-exec inside nix-shell for build dependencies
+if [ -f "$(dirname $0)/../.nixos" ] && [ -z "$IN_NIX_SHELL" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)